File: kalkulator/kalkulator.py
def dodawanie(a, b):
    return a + b

def odejmowanie(a, b):
    return a - b

def mnozenie(a, b):
    return a * b

def dzielenie(a, b):
    if b != 0:
        return a / b
    else:
        return "Nie można dzielić przez zero!"

print("Witaj w kalkulatorze!")

while True:
    print("Wybierz operację:")
    print("1. Dodawanie")
    print("2. Odejmowanie")
    print("3. Mnożenie")
    print("4. Dzielenie")
    print("5. Wyjście")

    wybor = input("Twój wybór: ")

    if wybor == "5":
        print("Do widzenia!")
        break

    liczba1 = float(input("Podaj pierwszą liczbę: "))
    liczba2 = float(input("Podaj drugą liczbę: "))

    if wybor == "1":
        wynik = dodawanie(liczba1, liczba2)
        print("Wynik dodawania:", wynik)
    elif wybor == "2":
        wynik = odejmowanie(liczba1, liczba2)
        print("Wynik odejmowania:", wynik)
    elif wybor == "3":
        wynik = mnozenie(liczba1, liczba2)
        print("Wynik mnożenia:", wynik)
    elif wybor == "4":
        wynik = dzielenie(liczba1, liczba2)
        print("Wynik dzielenia:", wynik)
    else:
        print("Nieprawidłowy wybór. Spróbuj ponownie.")

File: kalkulator/kalkulator.java
import java.util.Scanner;

public class kalkulator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Podaj pierwszą liczbę: ");
        double liczba1 = scanner.nextDouble();

        System.out.print("Podaj drugą liczbę: ");
        double liczba2 = scanner.nextDouble();

        System.out.println("Wybierz operację:");
        System.out.println("1. Dodawanie");
        System.out.println("2. Odejmowanie");
        System.out.println("3. Mnożenie");
        System.out.println("4. Dzielenie");

        int operacja = scanner.nextInt();

        double wynik;

        switch (operacja) {
            case 1:
                wynik = liczba1 + liczba2;
                System.out.println("Wynik dodawania: " + wynik);
                break;
            case 2:
                wynik = liczba1 - liczba2;
                System.out.println("Wynik odejmowania: " + wynik);
                break;
            case 3:
                wynik = liczba1 * liczba2;
                System.out.println("Wynik mnożenia: " + wynik);
                break;
            case 4:
                if (liczba2 != 0) {
                    wynik = liczba1 / liczba2;
                    System.out.println("Wynik dzielenia: " + wynik);
                } else {
                    System.out.println("Nie można dzielić przez zero!");
                }
                break;
            default:
                System.out.println("Nieprawidłowa operacja!");
        }

        scanner.close();
    }
}